fedmsg_meta_fedora_infrastructure/tests/__init__.py

```diff
@@ -66,6 +66,7 @@
 from fedmsg_meta_fedora_infrastructure.tests.pdc import *
 from fedmsg_meta_fedora_infrastructure.tests.mbs import *
 from fedmsg_meta_fedora_infrastructure.tests.centos_ci import *
+from fedmsg_meta_fedora_infrastructure.tests.waiverdb import *
```

Comment:
I recommend against this style of importing, as it's easy to end up with clashes, and even outside of clashes you can end up with a very polluted namespace. Explicit is better than implicit.